apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -70,12 +73,26 @@ li.ui-state-highlight {
   margin-bottom: 5px;
 }
 
+.pl-block-horizontal > .list-group{
+  overflow-x: scroll;
+}
+
+.pl-block-horizontal >  ul > li,
+.pl-block-horizontal > .list-group {
+  display: inline-block !important;
+  float: none !important;
+  white-space: nowrap;
```

Comment:
I think you actually want `display: inline` and `float: left` here. That solves the issues you were having with placeholders showing in the wrong place in some cases.

Also, get rid of the `!important`s

Reply:
this does make it so that if the blocks don't fit on one line, they go to the next line instead of making it scroll. If you want it to scroll, you should tweak `overflow` in some way, I think. 

It might be nice in the future to have it configurable between scrolling, or flowing on to the next line. But don't worry about that for now, just set it up the way you like it and we can make it configurable later if people want it.

Reply:
Swapping over to `display: block` and `float: left` did the trick. The debug on the browser was saying due to the use of float it was automatically switching to `display: block` . That said, I can't quite figure out how to get scrolling working with this configuration.

Reply:
Couldn't get scrolling working when using `display: inline` but I did finally find an attribute that seems to have fixed the alignment issues.